### DISCUSSION TITLE
How to work with cloud hosted repositories?

### DISCUSSION THREAD

**Discussion by davidtbo** [gollum/gollum#1985] [Ideas]
I see your instructions for running in docker, but what if you already have another hosted git repo? How would I use this with AWS CodeCommit? Assume it works as any other private git repo. 

Thanks!
Dave

**Comment by dometto** [comment]
There's currently no way to do this as a local repository is expected. However, the rugged and/or jgit backend libraries may already support cloud backed git repositories. If so, support for this would be a very welcome feature.

For now, a workaround could be setting up a local repository that syncs with an AWS remote (assuming CodeCommit can be accessed via HTTPS or SSH).

(As a side note: development on gollum has been very slow for various reasons, but I'm personally looking forward to getting back to it at the end of July -- sorry to anyone who has been waiting for the new release)

**Comment by davidtbo** [comment]
I hear about "too busy for side projects!" :) If I have time to work on it
I would be happy to contribute, but we'll see if I go with this one or a
different solution that will run on S3 & Lambda, & possibly noSQL.
